docs: add a Customize your template series under Get started (#26712)
coder/coder: docs/tutorials/quickstart/install-tools-with-mise.md

Author: The spirit of this page is completely off. It _is_ possible to use Homebrew with Coder workspaces. It just requires some additional configuration. The whole purpose of this document is to advertise that installing anything outside of the home directory in a workspace doesn't persist on restarts, and to link to persistent and ephemeral restarts. This PR has the code we use in dogfood to show how to install both Homebrew and mise: https://github.com/coder/coder/pull/24618

Author: From Coder Agents: Rewrote the page's spirit in 39b5f58e76. It no longer treats Homebrew as the wrong fit; it teaches that anything installed outside /home/coder is rebuilt away unless the template persists its location, using Homebrew vs mise as the contrast. The "make Homebrew persist" option points at the dogfood approach of mounting /home/linuxbrew, and the page links Resource persistence. I used PR #24618 as the source of truth for that mechanism.

# Install your own command-line tools

> This guide is for a developer who finished [Launch your first workspace](./launch-workspace.md)

Author: Same nit

Author: From Coder Agents: Direct-address intro here too, in 39b5f58e76.

and wants their favorite command-line tools in every workspace,
without editing the template.

The Quickstart template installs system languages through the **Programming Languages** parameter,

> but it does not carry the small command-line tools you reach for by habit,

Author: ```suggestion
but it does not carry the small command-line tools you may often use,
```

Author: From Coder Agents: Applied your wording in 39b5f58e76: "the small command-line tools you may often use".

such as [`bat`](https://github.com/sharkdp/bat) or [`ripgrep`](https://github.com/BurntSushi/ripgrep).
In this guide, you install one of those tools with [mise](https://mise.jdx.dev/),
watch the laptop habit of `brew install` fail first,

> and confirm the tool survives a workspace rebuild.

Author: I think this sentence should read, "In this guide, you'll install both Homebrew and Mise for installing tools and learn about persisting tool installations across workspace restarts."

Author: From Coder Agents: Reworked the page around exactly this in 39b5f58e76. The intro now says you install both Homebrew and mise and learn which installs survive a restart. I kept "mise" lowercase to match the project's own branding at mise.jdx.dev.

> [!NOTE]
> This guide works inside a running workspace from the Quickstart template.
> Unlike the other guides in this part,
> it does not edit the template,

> > so you do not need the template open for editing.

Author: From Coder Agents: I don't think that's quite right. The Quickstart template itself needs a direct update for Homebrew formulae to survive a workspace restart, and we're tracking that fix as a separate follow-up.

Author: From Coder Agents: You're right, and I removed the misleading note in 39b5f58e76. mise needs no template change because it installs into /home/coder, but making Homebrew persist does require a template change: mounting its /home/linuxbrew prefix on a volume, as the dogfood template does. The new "What just happened" section says this and links Resource persistence.

## What you'll do

> - ✅ Install a command-line tool into your workspace.

Author: ```suggestion
- ✅ Install command-line tools with [Homebrew](https://brew.sh/) and [Mise](https://mise.jdx.dev/) into your workspace.
```

Author: From Coder Agents: Applied in 39b5f58e76 (kept "mise" lowercase per the project's branding).

- ✅ Learn why `brew install` fails in the Quickstart workspace.
- ✅ Keep the tool through a workspace rebuild.

## What persists in a workspace

A Quickstart workspace keeps your home directory, `/home/coder`, on a persistent volume.
Everything outside `/home/coder` comes from the workspace image,

> and Coder rebuilds it from that image every time the workspace starts.

Author: Don't ruin the surprise this early. Start by getting users to install Homebrew and Mise through the recommended installation methods. Then, install `bat` through Mise or Homebrew, and then install Ripgrep through the other package manager.

Then, have them reboot the workspace, and watch one of the programs persist and the other to disappear. And _then_ you can explain why the failure occurred and how to correct it.

Author: From Coder Agents: Restructured to this flow in 39b5f58e76: install both Homebrew and mise, install ripgrep with Homebrew and bat with mise, confirm both work, then restart and compare. Only after the restart, when bat persists and rg and brew are both gone, does the page explain why and how to make either one stay. I verified the mechanics in codercom/enterprise-base:ubuntu: brew installs to /home/linuxbrew and bat via mise lands under /home/coder.

A tool survives a rebuild only when two things are true:

- The tool installs into `/home/coder`.
- Your shell finds the tool through a file in `/home/coder`, such as `.bashrc`.

[mise](https://mise.jdx.dev/) is a version manager that meets both conditions.
It installs tools under `~/.local/share/mise` and activates from `~/.bashrc`,
both inside your home directory.

## Step 1: Watch a laptop habit fail

On a laptop you might install a tool with [Homebrew](https://brew.sh/).
Open a terminal in your workspace and try it:

```sh
brew install bat
```

The command fails:

```text
brew: command not found
```

> Homebrew is the wrong fit here for two reasons:

Author: I wouldn't say Homebrew is the wrong fit here. I would say, "In your workspace, `brew` commands throw an error for the following reasons:"

Author: From Coder Agents: Reconciled per your clarification in 39b5f58e76. brew works while the workspace runs; the error appears after the restart, when /home/linuxbrew is gone and both brew and rg report "command not found". The page uses that as the post-restart proof rather than leading with a failure.
